lessons/pydata/pandas_core/index.ipynb

```diff
@@ -0,0 +1,3402 @@
+{
```

Comment:
<p>Obrázek se tu nenačte. V reviewnb.com ani na githubu. Možná jenom chyba náhledů.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Lokálně se zobrazí. Vyrenderovaný by snad taky měl. Je to prostá relativní cesta.</p>